addons/bus/controllers/main.py

```diff
@@ -38,3 +38,7 @@ def poll(self, channels, last, options=None):
         if request.registry.in_test_mode():
             raise exceptions.UserError(_("bus.Bus not available in test mode"))
         return self._poll(request.db, channels, last, options)
+
+    @route('/longpolling/im_status', type="json", auth="public", cors="*")
```

Comment:
auth="public" ?

Reply:
removed `auth="public"` and `cors="*"`, since this route is only called from `/web` url